### PR TITLE
Feature/emission targets

### PR DESCRIPTION
**Pull request by tsubik**
- emission targets importer
- emission targets endpoint /api/v1/emission_targets

`bundle exec rake db:migrate`
`bundle exec rake db:admin_boilerplate:create`

`bundle exec rake emission_targets:import`

### CODE REVIEW

File: app/models/concerns/generic_to_csv.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+require 'csv'
```
Comment (agnessa) [inline app/models/concerns/generic_to_csv.rb]: This module is available from the climate_watch_engine gem from version 1.3.2, so we could avoid duplicating it here (unless you're making changes to it?)

File: spec/controllers/api/v1/emission_targets_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+describe Api::V1::EmissionTargetsController, type: :controller do
```
Comment (agnessa) [inline spec/controllers/api/v1/emission_targets_controller_spec.rb]: Don't know why, but unless I add `, format: :json`, all 3 controller specs fail, is it just me? Noted there's no travis configured, I'll get that sorted.

**Comment by tsubik** [comment]
Thanks @agnessa looks like I forgot to run specs after final changes :)

File: app/models/concerns/generic_to_csv.rb (hunk 1 above)
Comment (tsubik) [reply app/models/concerns/generic_to_csv.rb]: Done

File: spec/controllers/api/v1/emission_targets_controller_spec.rb (hunk 2 above)
Comment (tsubik) [reply spec/controllers/api/v1/emission_targets_controller_spec.rb]: :+1:

**Pull request merged**